main.js

```diff
@@ -0,0 +1,236 @@
+const CurrentWeather = function () {
+
+  // Property name to be saved in local storage when a user picks a default location
+  const STORAGE_ID = 'Weather';
+
+  // Save the weather object, which contains both current and forecast data, in local storage
+  let saveToLocalStorage = function () {
+    localStorage.setItem(STORAGE_ID, JSON.stringify(weather));
+  }
+
+  // Get the weather object, which contains both current and forecast data, from local storage
+  let getFromLocalStorage = function () {
+    return JSON.parse(localStorage.getItem(STORAGE_ID) || '{}');
+  }
+
+
+  /*
+    Weather object will hold current weather data based on the user's query.
+    Object includes the current weather and a separate array to contain an object for
+    each of the five days in the forecast. Data is pulled from local storage, if a default
+    location has been set by the user.
+  */
+  let weather = getFromLocalStorage();
+
+  // Empty the weather object when conducting a new search query
+  let clearWeather = function () {
+    weather = {};
+  }
+
+  // Replace the first character of each word in the conditions string with its uppercase counterpart
+  let capitalizeConditionsString = (string) => {
+    return string.replace(/\w\S*/g, (txt) => txt.charAt(0).toUpperCase() + txt.substr(1));
+  }
+
+  /*
+    After getting the current conditions and forecast data into our weather data structure,
+    remove the old weather information from the page, and append our new data on the page.
+  */
+  let renderWeather = function () {
+    // Remove old data from view, if any
+    $('#current-weather').empty();
+    $('#five-day-weather').empty();
```

Comment:
Why not reset the body class to a default value here too?  That's part of the "clear" functionality here.